src/runtime_src/core/pcie/driver/linux/xocl/subdev/command_queue.c

```diff
@@ -545,6 +595,12 @@ static int command_queue_probe(struct platform_device *pdev)
 		}
 	}
 
+	err = sysfs_create_group(&pdev->dev.kobj, &cmd_queue_attr_group);
+	if (err) {
+		xocl_err(&pdev->dev, "create ert_cq sysfs attrs failed: %d", err);
```

Comment:
later on, we might have more and more sysfs node for other purpose. Should we print "Create command queue sysfs node group failed" here?

Reply:
ert_cq will be the sub-dev name. Not just the argument.